lib/Sapi.php

```diff
@@ -71,6 +71,11 @@ static function sendResponse(ResponseInterface $response) {
         $body = $response->getBody();
         if (is_null($body)) return;
 
+        if (is_callable($body)) {
+            $body();
```

Comment:
shouldn't we pass the $response object into the callback so it can handle the case of existing Content-Length header etc? (like wie do for non-callback bodies a few lines below)